.github/workflows/build_and_test.yml

```diff
@@ -532,27 +532,27 @@ jobs:
       - name: Report Selenium
         if: always()
         run: |
-          cd test/reporter
+          cd .github/reporter
           npm install
           SUITE=${{ matrix.suite }} node report.js
 
       - name: docker logs
         if: always()
-        run: test/docker_logs.sh
+        run: test/scripts/docker_logs.sh
 
       - name: mediawiki logs
         if: always()
         continue-on-error: true
         run: |
-          ls -lahr test/log test/log/*
-          tail -n +1 test/log/*/*.log
+          ls -lahr test/suite-config/${{ matrix.suite }}/results test/suite-config/${{ matrix.suite }}/results/*
+          tail -n +1 test/suite-config/${{ matrix.suite }}/results/*.log
 
       - name: Archive docker test artifacts
         uses: actions/upload-artifact@v3
         if: always()
         with:
           name: TestArtifacts
-          path: test/log
+          path: test/suite-config/${{ matrix.suite }}/results
```

Comment:
Note: This probably needs to be changed to `test/suite-config/**/results` to get the right folders in the TestArtifacts archive. Current archive is a flat directory.